bin/devenv

```diff
@@ -18,16 +18,21 @@ elif [[ -z ${cmd} || ${cmd} = help ]]; then
   display "Usage: ${0##*/} [command]
 
 Description:
-devenv management tool
+    Tool to manage development environment in console
+
+Variables:
```

Comment:
In the second commit 882c52f4d3b2437bfa2609f3ca9f703164af86e1 I add the environment variables to the help message.